### ISSUE TITLE
Regression: Magic comments in v8.0.3

### ISSUE THREAD

**Issue by comicchang** [James-Yu/LaTeX-Workshop#1611]
**Disable all the other extensions except for LaTeX Workshop, and check that you still see this issue. [Required]**

You still see this issue?: Yes

**Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.**

**Describe the bug [Required]**
commit https://github.com/James-Yu/LaTeX-Workshop/commit/af4506d519a2a2acd898aaf8104730dfa75225f5 broken some part of magic comments.

**To Reproduce**
1. make a empty folder, add some tex files
```
main.tex
data/1.tex
...
```
2. add magic comments into files   
in file main.tex   
```
% !TEX program = latexmk
% !TEX options = -xelatex -synctex=1 -interaction=nonstopmode -file-line-error "%DOC%"
```

in file 1.tex   
```
% !TEX root = ../main.tex
```
3. edit and save 1.tex
4. magic comments in main.tex not working, and latex workshop use latexmk with pdflatex, which will cause errors. this was working great pre version v8.0.3

**Expected behavior**
when saving 1.tex, it will looks up magic comment in main.tex, and use latexmk with xelatex to compile everything.

**Logs [Required]**
no

<details>
<summary>LaTeX Workshop Output [Required]</summary>

</details>

<details>
<summary>Developer Tools Console [Required]</summary>

</details>

**Screenshots**

**Desktop [Required]:**
 - OS: MacOS 10.14.6
 - VS Code version: 1.37.1
 - Extension version: 8.0.3

**Are you using LaTeX Workshop with VS Code Remote?**

No

**Additional context**

**Comment by James-Yu** [comment]
I cannot reproduce the issue.

**Comment by James-Yu** [comment]
```
[17:40:21] BUILD command invoked.
[17:40:21] Found root file by magic comment: c:\Users\James\Desktop\test\main.tex
[17:40:21] Root file remains unchanged from: c:\Users\James\Desktop\test\main.tex.
[17:40:21] Building root file: c:\Users\James\Desktop\test\main.tex
[17:40:21] Build root file c:\Users\James\Desktop\test\main.tex
[17:40:21] Found TeX program by magic comment: latexmk
[17:40:21] Found TeX options by magic comment: -xelatex -synctex=1 -interaction=nonstopmode -file-line-error "%DOC%"
[17:40:21] Recipe step 1: latexmk, --max-print-line=10000,-xelatex -synctex=1 -interaction=nonstopmode -file-line-error "c:/Users/James/Desktop/test/main"
[17:40:21] LaTeX build process spawned. PID: 6776.
[17:40:22] Recipe of length 1 finished. PID: 6776.
[17:40:22] Successfully built c:\Users\James\Desktop\test\main.tex.
```
![image](https://user-images.githubusercontent.com/4210342/63255506-800ada80-c2a8-11e9-8a8f-bddcf4332797.png)
![image](https://user-images.githubusercontent.com/4210342/63255515-87ca7f00-c2a8-11e9-8696-b6eb300a3547.png)

**Comment by comicchang** [comment]
sorry, it seems something wrong in my config.

**Comment by comicchang** [comment]
sorry, my mistake, it seems that the regression is fixed in v8.0.3 rather than introduced in v8.0.3, and since its already fixed, I'll just leave this issue closed.
